newapi/ooniapi/citizenlab.py

```diff
@@ -411,14 +422,12 @@ def check_url(url):
         raise BadURL()
 
 
-def validate_entry(entry: List[str]) -> None:
-    url, category_code, category_desc, date_str, user, notes = entry
-    check_url(url)
-    if category_code not in CATEGORY_CODES:
+def validate_entry(entry: Dict[str, str]) -> None:
+    check_url(entry["url"])
+    if entry["category_code"] not in CATEGORY_CODES:
         raise BadCategoryCode()
-    if category_desc != CATEGORY_CODES[category_code]:
```

Comment:
You should either leave this check of the category_desc, or uncomment the line above where you override it.

Reply:
(I'm aware, the code is WIP)